backend/src/controllers/utils/billit/createorderrequestfactory/create-order-request.factory.ts

```diff
@@ -5,7 +5,7 @@ import {fromClient as createCustomerFromClient} from './customer.factory';
 import {fromInvoice as createOrderLinesFromInvoice} from './orderlinesfactory';
 import {fromInvoice as createPaymentReferenceFromInvoice} from './payment-reference.factory';
 
-const InvoiceExpirationInDays = 14;
+const InvoiceExpirationInDays: number = 14;
```

Comment:
dit is impliciet, zonder de `: number` is die constante ook een strongly typed number